smartsim/error/errors.py

```diff
@@ -149,3 +149,11 @@ class UnproxyableStepError(TelemetryError):
 
 class SmartSimCLIActionCancelled(SmartSimError):
     """Raised when a `smart` CLI command is terminated"""
+
+
+class PreviewException(SSUnsupportedError):
+    """Raised when a part of preview isn't supported by SmartSim yet"""
```

Comment:
- remove `yet` from description (it implies a feature will be supported on some unknown timeline)
- Consider renaming to indicate purpose. `PreviewException` is pretty generic while the description is specifically about missing features.

Reply:
what about previewunsupportedexception?